Library/Formula/fmpp.rb

```diff
@@ -0,0 +1,16 @@
+class Fmpp < Formula
+  homepage "http://fmpp.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/fmpp/fmpp/0.9.15/fmpp_0.9.15.tar.gz"
+  sha256 "b893451b5450a7f35fe680e934f6903ec8143d88959dcfca5d17467fbe4142f9"
+
+  depends_on :java
+
+  def install
+    libexec.install "lib"
+    bin.write_jar_script libexec/"lib/fmpp.jar", "fmpp", "-Dfmpp.home=#{libexec} $FMPP_OPTS $FMPP_ARGS"
+  end
+
+  test do
+    system "fmpp", "-h"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!